app/models/user.rb

```diff
@@ -1,9 +0,0 @@
-class User < ApplicationRecord
-  has_many :posts, foreign_key: :author_id
-  has_many :comments, foreign_key: :author_id
-  has_many :likes, foreign_key: :author_id
-
-  def most_recent_3_posts
-    Post.where(author_id: id).order(created_at: :desc).limit(3)
-  end
-end
```

Comment:
- [ ] I think you have deleted all the association and model work that was needed in this milestone and your application. Please fix this.